### PR TITLE
Fix issue 30 : Save error code non existent command

### PR DESCRIPTION
**Pull request by HugFes**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Fixed issue   | #30 
| License       | MIT

Save last error code for non existent function when is running at  last command

### CODE REVIEW

**Review by oallain — APPROVED** [review]
Thanks @HugFes :rocket:

**Comment by oallain** [comment]
@HugFes Can you check message form CLAassistant https://github.com/synolia/SyliusSchedulerCommandPlugin/pull/31#issuecomment-710004483 ?

**Comment by HugFes** [comment]
Thanks for your review.

Yes I see it and it's done

File: tests/PHPUnit/Command/SynoliaSchedulerRunCommandTest.php [hunk 1]
```diff
@@ -78,6 +80,52 @@ public function testExecuteWithANewCommandThatItMarkedAsExecuteNow(): void
         self::assertStringNotContainsString('Nothing to do', $commandTester->getDisplay());
     }
 
+    /**
+     * @group toto
```
Comment (Jibbarth) [inline tests/PHPUnit/Command/SynoliaSchedulerRunCommandTest.php:84]: What is this group ?

Comment (oallain) [reply tests/PHPUnit/Command/SynoliaSchedulerRunCommandTest.php:84]: I see it, and i'll remove it, thanks for report.

Comment (HugFes) [reply tests/PHPUnit/Command/SynoliaSchedulerRunCommandTest.php:84]: Oh sorry I forgot to remove it. It just for me when I debug this test. 
Thanks for repport @Jibbarth

**Pull request merged**